torch/__init__.py

```diff
@@ -141,25 +141,38 @@ def _preload_cuda_deps():
     """Preloads cudnn/cublas deps if they could not be found otherwise."""
     # Should only be called on Linux if default path resolution have failed
     assert platform.system() == 'Linux', 'Should only be called on Linux'
-    cublas_path = None
-    cudnn_path = None
+
+    cuda_libs: Dict[str, str] = {
+        'cublas': 'libcublas.so.11',
+        'cudnn': 'libcudnn.so.8',
+        'cuda_nvrtc': 'libnvrtc.so.11.2',
+        'cuda_runtime': 'libcudart.so.11.0',
+        'cuda_cupti': 'libcupti.so.11.7',
+        'cufft': 'libcufft.so.10',
+        'curand': 'libcurand.so.10',
+        'cusolver': 'libcusolver.so.11',
+        'cusparse': 'libcusparse.so.11',
+        'nccl': 'libnccl.so.2',
```

Comment:
Perhaps I've missed a prep change, but I thought PyTorch still builds nccl from source, isn't it?

Reply:
@malfet This is nccl change in builder that sets a dynamic version: https://github.com/pytorch/builder/pull/1184